tests/app/settings.py

```diff
@@ -17,6 +17,7 @@ def env(name, default=None):
 
     'taggit',
     'modelcluster',
+    'captcha',
```

Comment:
We can remove this since it is an optional dependency for usage and its up to end-developer to use this or not.